src/stepfunctions/steps/compute.py

```diff
@@ -12,8 +12,31 @@
 # permissions and limitations under the License.
 from __future__ import absolute_import
 
+from enum import Enum
 from stepfunctions.steps.states import Task
 from stepfunctions.steps.fields import Field
+from stepfunctions.steps.integration_resources import IntegrationPattern, get_service_integration_arn
+
+LAMBDA_SERVICE_NAME = "lambda"
+GLUE_SERVICE_NAME = "glue"
+ECS_SERVICE_NAME = "ecs"
+BATCH_SERVICE_NAME = "batch"
+
+
+class LambdaApi(Enum):
+    Invoke = "invoke"
+
+
+class GlueApi(Enum):
```

Comment:
not blocking: also agree that these don't **need** to be classes. 

If you do keep them as classes, we should include docstrings that explain what they are and their purpose :)